yb-voyager/cmd/common.go

```diff
@@ -392,8 +392,7 @@ func CreateMigrationProjectIfNotExists(dbType string, exportDir string) {
 
 	setSourceDbType(dbType, projectDirPath)
 
-	schemaObjectList := utils.GetSchemaObjectList(dbType)
-	utils.ApplyExportSchemaObjectListFilter(&schemaObjectList, source.ExportObjects)
+	schemaObjectList := source.ExportObjectTypesList
```

Comment:
Why not directly refer the `source.ExportObjectTypesList` in the for loop?

Reply:
Done